src/pip/_internal/index.py

```diff
@@ -673,6 +673,10 @@ def create(
             abi=abi,
             impl=implementation,
         )
+        logger.debug("Valid tags:")
+        with indent_log():
+            for tags in valid_tags:
+                logger.debug(" ".join(tags))
```

Comment:
A few comments on this: I would do this in a single debug log message, not use `indent_log()` (since it's a single message), join on `', '` rather than `' '`, and use the `format_tag()` function added for this purpose and that is used elsewhere in `index.py` for logging tags: https://github.com/pypa/pip/blob/107258da0b23acf44c18377387a1d4b21657ef8d/src/pip/_internal/wheel.py#L666-L673